bundle/manifests/kataconfiguration.openshift.io_kataconfigs.yaml

```diff
@@ -111,9 +111,14 @@ spec:
                 description: KataMonitorImage is used to specify the container image
                   used for kata-monitor
                 type: string
+              logLevel:
+                default: info
+                description: Sets log level on kata-equipped nodes.  Valid values are the same as for `crio --log-level`.
+                type: string
             required:
             - checkNodeEligibility
             - kataMonitorImage
+            - logLevel
```

Comment:
when we make it required we should bump up the version of kataconfig

Reply:
@jensfr Are we fine bumping up the version, are there risks? I don't really have a strong opinion about making it required, I only did it because I saw `checkNodeEligibility` was required as well and because it slightly simplifies handling of the field in code.  For the user it doesn't seem to make much difference - if a required field has a default they don't have to specify it anyway.

Reply:
@pmores, we learnt it the hard way. Having it `required` will fail migration.  We'll have to make checkNodeEligibility and kataMonitorImage also optional to sync with downstream.  Or we bump up the version.